src/debug/inc/dacdbistructures.h

```diff
@@ -573,7 +573,6 @@ class MSLAYOUT FieldData
     bool            m_fFldIsStatic;           // true if static field, false if instance field
     bool            m_fFldIsRVA;              // true if static relative to module address
     bool            m_fFldIsTLS;              // true if thread-specific static
-    bool            m_fFldIsContextStatic;    // true if context-specific static
```

Comment:
Is this exposed to SOS somehow? Would this break SOS targeting older runtimes?

Reply:
I  believe the FieldData structure is not one of those where the layout needs to be preserved. I think that all such structures are in dacprivate.h.
As for the field itself, we have never supported context statics on CoreCLR, so the `m_fFldIsContextStatic` was always false and so the removal of the field usage basically reflects that.

Reply:
Sounds good.